metabase/metabase
# Add types to custom cypress commands for auth

Continuing https://github.com/metabase/metabase/pull/36813

Motivated by https://github.com/metabase/metabase/pull/36258#discussion_r1431879509

## Summary

Add types for custom commands like `cy.signInAsAdmin()` so we don’t have to use `ts-ignore`.

I only included the authentication commands for now, but it’s a start.

[Relevant Cypress docs here](https://docs.cypress.io/guides/tooling/typescript-support#Types-for-Custom-Commands)

## How to verify

No e2e specs in TS on master yet, but the fix is verified by [this line in another PR](https://github.com/metabase/metabase/pull/36258/files#diff-0edea2253acd2b512aade59ec3edb5019e5f6928cccd6c5d25bd0ceffbb5faf2R83), since the type-check passed and is rebased on this branch.
